prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.ts

```diff
@@ -25,6 +26,7 @@ export class EnrolleeOverviewComponent extends AdjudicationContainerComponent im
   public enrollee: HttpEnrollee;
   public enrolment: Enrolment;
   public enrolleeNavigation: EnrolleeNavigation;
+  public hideAdjudication: boolean;
```

Comment:
My only suggestion would be to make this `showAdjudication`, which by default will be false, and then let the data show it.  Currently as the page loads it should be hidden and then we decide to show it.